extensions/amp-form/amp-form.md

```diff
@@ -75,7 +75,108 @@ __required__
 
 Action must be provided, `https` and is non-cdn link (does **NOT** link to https://cdn.ampproject.org).
 
+**action-xhr**
+__optional__
+You can also provide an action-xhr attribute, if provided, the form will be submitted in an XHR fashion.
+
+This attribute can be the same or a different endpoint that action and has the same action requirement above.
+
+**Important**: Your XHR endpoints need to follow and implement [CORS Requests in AMP spec](https://github.com/ampproject/amphtml/blob/master/spec/amp-cors-requests.md).
+
 All other [form attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) are optional.
 
 ## Inputs
 Currently, `<input type=button>`, `<input type=file>`, `<input type=image>` and `<input type=password>` are not allowed. (This might be reconsidered in the future - please let us know if you require these and use cases).
+
+## Events
+`amp-form` exposes 3 events:
+**submit**
```

Comment:
Can you add a line before this line. The "submit" line is showing up at the end of the above line with the colon.

Might also want to consider making these three a bulleted list.